### DISCUSSION TITLE
Add button to change dark/light background in the project editor

### DISCUSSION THREAD

**Discussion by sfederici** [ScratchAddons/ScratchAddons#5708] [Ideas]
When editing costumes with very light colors (such as, for example, the costumes of sprite1 in project [https://scratch.mit.edu/projects/798179240](https://scratch.mit.edu/projects/798179240)) it is really difficult to spot the light lines against the "transparent" background. On the contrary, this is really easy if you switch to a dark theme in Scratch Addons settings. However, if you usually use a light theme, switching back and forth from light to dark and viceversa by means of the Scratch Addons settings page is really time consuming.

Proposal: add a button in the Scratch editor to switch from light to dark background and viceversa (or, better, from the background currently set in Scratch Addons settings -if any- to dark and viceversa).

**Comment by Secret-chest** [comment]
Optional, please.

**Reply by sfederici** [reply]
> Optional, please.

Being an addon, they can all be enabled optionally in the settings, aren't they? Or do you mean disabled by default? If so, I totally agree.

**Reply by Secret-chest** [reply]
> > Optional, please.
> 
> Being an addon, they can all be enabled optionally in the settings, aren't they? Or do you mean disabled by default? If so, I totally agree.

Wouldn't this be in the editor dark mode addon?

**Reply by sfederici** [reply]
Yes, I think this could be a good addition to the dark mode addon.

**Comment by CST1229** [comment]
TurboWarp has this for the paint editor specifically, that toggles only the paint editor's background.

**Reply by sfederici** [reply]
Excellent. This is exactly what I would like to use in Scratch too

**Comment by Samq64** [comment]
I refrenced this in a new issue so it's a little easier to keep track of: #5711
